Makefile

```diff
@@ -44,9 +45,9 @@ clean: .env stop
 	rm -rf coverage/ .coverage
 	rm -fr .docker-build
 
-.PHONY: migrate
-migrate: .env
-	docker-compose run web python manage.py migrate --run-syncdb
+.PHONY: setup
+setup: .env
+	docker-compose run web /app/bin/setup-services.sh
```

Comment:
`make setup` will wipe and recreate the db, so I don't think we need `migrate` anymore.